com.chartboost.mediation/Runtime/FullScreen/IChartboostMediationFullScreenAdOLD.cs

```diff
@@ -1,37 +1,31 @@
-using UnityEngine.Scripting;
-
 namespace Chartboost.FullScreen
 {
     /// <summary>
     /// Interface implemented by Chartboost Mediation fullscreen ads. 
     /// </summary>
-    public interface IChartboostMediationFullScreenAd
+    public interface IChartboostMediationFullScreenAdOld
```

Comment:
I guess you forgot to rename this back to `IChartboostMediationFullScreenAd` ?

Reply:
Well not really, we already have a new interface with the same name. I don't think anybody is referencing this interface anyways so it should not be an issue.

Reply:
Oh right, I didn't see the new interface earlier !